pincer/client.py

```diff
@@ -5,20 +5,26 @@
 
 import logging
 from asyncio import iscoroutinefunction, run, ensure_future
+from importlib import import_module
 from inspect import isasyncgenfunction
 from typing import Optional, Any, Union, Dict, Tuple, List
 
+from pincer.objects import AppCommand

```

Comment:
Change this to relative import

Reply:
If its even used :?

Reply:
used as a typehint at line 371 `to_remove: List[AppCommand] = []`

Reply:
Resolved with `TYPE_CHECKING`